### PR TITLE
Document installing extensions using full repo URL

### PR DESCRIPTION
**Pull request by wrslatz**
### Why:

Related to https://github.com/cli/cli/issues/4201. Making a contribution to `cli` repo as well with similar documentation.

### What's being changed:

Documents the ability to install extensions using a full repo URL. This is helpful for GitHub Enterprise users who may want to install extensions from public GitHub.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by wrslatz** [comment]
Here's the PR for the CLI: https://github.com/cli/cli/pull/4271

**Comment by ramyaparimi** [comment]
@wrslatz 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by wrslatz** [comment]
Thanks! My corresponding CLI PR had some cleanup made so we may want to double-check and align the content here.

**Comment by lecoursen** [comment]
Hi @wrslatz, this information will be a really great addition! Instead of adding a note, though, I'd like to rework the existing instructions instead. I can't leave this as a suggested comment because it's too many lines away from the changes lines 😭 but can you update the PR to take this type of approach?

> To install an extension, use the `extensions install` subcommand. Replace the `repo` parameter with the repository of the extension. You can use the full URL, such as `https://github.com/octocat/gh-whoami`, or just the owner and repository, such as `octocat/gh-whoami`. If the full URL is not used, `gh` will install the extension using the hostname to which `gh` is currently authenticated. Using the full repository URL allows GitHub Enterprise Server users to install extensions from GitHub.com.
>
> ```shell
>gh extension install <em>repo</em>
> ```

Thanks!

**Comment by wrslatz** [comment]
> but can you update the PR to take this type of approach?

Yep! Happy to make that adjustment.

Should I also document installing a local extension, like was added in https://github.com/cli/cli/pull/4271? Or is that a more advanced usage / specific to extension development where it's best kept separate?

**Comment by wrslatz** [comment]
> Yep! Happy to make that adjustment.

I've made the adjustments, @lecoursen 👍🏻 thanks for the feedback!

**Comment by lecoursen** [comment]
> Should I also document installing a local extension, like was added in [cli/cli#4271](https://github.com/cli/cli/pull/4271)? Or is that a more advanced usage / specific to extension development where it's best kept separate?

I think that's a great idea actually!

**Comment by wrslatz** [comment]
> I think that's a great idea actually!

I'll try to work that in real quick

**Comment by wrslatz** [comment]
Done! How does that look?

File: content/github-cli/github-cli/using-github-cli-extensions.md [hunk 1]
```diff
@@ -27,10 +27,14 @@ You can find extensions by browsing [repositories with the `gh-extension` topic]
 
 ## Installing extensions
 
-To install an extension, use the `extensions install` subcommand. Replace the `owner/repo` parameter with the name of the extension, such as `octocat/gh-whoami`.{% ifversion ghes %} If the extension is on {% data variables.product.prodname_ghe_server %}, also include the hostname, such as `https://ghe.io/octocat/gh-whoami`.{% endif %}
+To install an extension, use the `extensions install` subcommand. Replace the `repo` parameter with the repository of the extension. You can use the full URL, such as `https://github.com/octocat/gh-whoami`, or just the owner and repository, such as `octocat/gh-whoami`.
+
+If the owner and repository are used, `gh` will install the extension using the hostname to which `gh` is currently authenticated. The full URL format is useful when installing extensions from a different host. For example, users on {% data variables.product.prodname_ghe_server %} should use the full repository URL to install extensions from GitHub.com or any other host.
```
Comment (lecoursen) [inline content/github-cli/github-cli/using-github-cli-extensions.md:32]:
```suggestion
If the owner and repository are used, `gh` will install the extension using the hostname to which `gh` is currently authenticated. The full URL format is useful when installing extensions from a different host. For example, users on {% data variables.product.prodname_ghe_server %} should use the full repository URL to install extensions from {% data variables.product.prodname_dotcom_the_website %} or any other host.
```

**Review by lecoursen — COMMENTED** [review]
This is fantastic!! I left one suggestion to use a product name variable but will merge this as soon as tests pass after applying it. ⚡

**Review by lecoursen — APPROVED** [review]

**Pull request merged**